es61x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala

```diff
@@ -107,6 +108,10 @@ class ReadonlyRestPlugin(s: Settings, p: Path)
     .load(environment.configFile)
     .map(_.fold(e => throw new ElasticsearchException(e.message), identity))
     .runSyncUnsafe(timeout)(Scheduler.global, CanBlock.permit)
+  private val fipsConfig = FipsConfiguration
```

Comment:
maybe we can create a helper to run it like this:
```
RorSsl
    .load(environment.configFile)
    .getOrThrowElasticsearchException
```

Reply:
We could, but currently we are throwing `ElasticsearchException` which is not accessible from the core and creating this helper in each module probably won't make sense. It would be easy to make such helper if we could throw some non elastic exception